ibm_catalog.json

```diff
@@ -0,0 +1,110 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-cloudant",
+      "label": "IBM Cloudant module",
```

Comment:
This is a DA, not a module. I'm thinking we just name it `IBM Cloudant`. We may be asked to change this when we go to publish publicly but its fine for now

Reply:
How about `IBM Cloudant Dedicated`?

Reply:
nevermind - read your next comment - saying that we want to keep this generic. Hence no "dedicated" in the name of DA